lib/helpers.js

```diff
@@ -1,5 +1,5 @@
 cl = function(something){
```

Comment:
Well I had this idea of a general log helper that you could toggle on and off through a `debug` flag, to make it easier to debug issues. But I don't know if that's a good idea, and anyway you're right that it's not used. So it might be best to remove it.